### DISCUSSION TITLE
How to restrict search scope to specific page?

### DISCUSSION THREAD

**Discussion by msoutopico** [squidfunk/mkdocs-material#4965] [General]
Hi there, 

I'm authoring some guides in this domain: https://capstanlqc.github.io/omegat-guides/

However, each user receives the link to the specific guide that they need, e.g. https://capstanlqc.github.io/omegat-guides/translation/ for a translator, https://capstanlqc.github.io/omegat-guides/verification/ for a verifier, etc. 

Currently, if the user runs a search from the search box on the right top search box, they get results from all the guides in the site. 

Would it be possible to restrict the scope of the search so that a user only gets results from pages that have a certain base URL? (so that a translator doesn't get results from pages for, say, a verfier, and vice versa). 

I hope my question is clear.

Thanks a lot.
Cheers, Manuel

**Comment by squidfunk** [comment]
Currently not possible. However, you could build separate MkDocs sites and link the together to achieve the same thing.

**Reply by msoutopico** [reply]
Thanks. Is that (limite the indexing scope) a custom development you could consider?

Another question: I guess to build separate MkDocs sites I would need one separate Github repo for each site, correct? 

Thank you, Martin.

**Reply by squidfunk** [reply]
Maybe, I'll consider it next time I work on search 😊 You can use a single repository with subfolders with a `mkdocs.yml` each.

**Reply by msoutopico** [reply]
> Maybe, I'll consider it next time I work on search blush You can use a single repository with subfolders with a `mkdocs.yml` each.

Thanks, that is helpful!

**Reply by squidfunk** [reply]
Please see the announcement in https://github.com/squidfunk/mkdocs-material/issues/6307.
